net/mox-pkcs11/Makefile

```
@@ -0,0 +1,50 @@
+#
+## Copyright (C) 2020, 2025 CZ.NIC z.s.p.o. (https://www.nic.cz/)
+#
+## This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+# #
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mox-pkcs11
+PKG_VERSION:=2.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/mox-pkcs11.git
+PKG_MIRROR_HASH:=424b5247288310c6a71b7babed201dc153c58fca73241530254d2cbc84f1b4ef
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+
+PKG_MAINTAINER:=Tomáš Macholda <tomas.macholda@nic.cz>
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_LICENSE_FILES:=LICENSE.txt
+
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/mox-pkcs11
+  TITLE:=MOX PKCS11
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=SSH
+  DEPENDS:=@(TARGET_mvebu_cortexa53_DEVICE_ripe_atlas-v5||TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox) +libopenssl +libkeyutils
+  URL:=https://gitlab.nic.cz/turris/mox-pkcs11
+endef
```

```suggestion
define Package/mox-pkcs11
  SECTION:=utils
  CATEGORY:=Utilities
  TITLE:=MOX PKCS11 module
  DEPENDS:=@(TARGET_mvebu_cortexa53_DEVICE_ripe_atlas-v5||TARGET_mvebu_cortexa53_DEVICE_cznic_turris-mox) +libopenssl +libkeyutils
  URL:=https://gitlab.nic.cz/turris/mox-pkcs11
endef
```

Just reordering and moving this to Utilities.